int32 motor_current_speed
int32 motor_current_direction
int32 linear_current_speed
int32 linear_current_direction

uint16 temp1
uint16 temp2

float32 motor_current
float32 linear_current

bool switch1_out
bool switch2_out

bool homing_needed
